model/src/main/kotlin/config/CliConfiguration.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2017-2020 HERE Europe B.V.
```

Comment:
The years should be just 2020.

Reply:
Done. But I thought we agreed to keep it like this for simplicity.